### ISSUE TITLE
Think of more intuitive ini configuration structure

### ISSUE THREAD

**Issue by SlothOfAnarchy** [stuvusIT/php-fpm#23]
Something like 
```yml
php_fpm_php_ini_values:
  PHP:
   - extension: php_mysqli
   - extension: php_ldap
```

**Comment by dasJ** [comment]
Automatically bumping this issue because it was inactive for 35 days.

**Comment by dasJ** [comment]
Automatically bumping this issue because it was inactive for 42 days.

**Comment by dasJ** [comment]
Automatically bumping this issue because it was inactive for 85 days.

**Comment by dasJ** [comment]
Automatically bumping this issue because it was inactive for 104 days.

**Comment by Scriptkiddi** [comment]
this is not possible

**Comment by SlothOfAnarchy** [comment]
Did you test this? I bet it is possible using `with_dict`

**Comment by Scriptkiddi** [comment]
not really since you are editing an ini file you need to know where an entry belongs to its the minimum the ansible ini module takes to add or remove an entry
  - section: PHP
    option: extension
    value: php_ldap

**Comment by SlothOfAnarchy** [comment]
Yes, but a smoother configuration would still be possible.
```yml
iniconfig:
  section:
    key1: value1
    key2: value2
```
You get section as the key in the dict iteration of php_fpm_php_ini_values
Then you iterate over the value (a dict) and write each key/value as ini option/value. 

I know it may get bloated but it's about easier use in configurations

**Comment by Scriptkiddi** [comment]
should be fixed by #34
